src/wrappers.hpp

```diff
@@ -716,7 +716,6 @@ void Symmetrize(Eigen::PlainObjectBase<Derived>& m)
 inline double FSThrow(const std::string& s)
 {
    throw PhysicalError(s);
-   return 0.;
```

Comment:
Just a note: This return statement has originally been added to suppress a strange compiler warning.

Reply:
reverted